src/components/common/ListNameTranslations.vue

```diff
@@ -19,15 +19,15 @@
       </v-col>
 
       <!-- Actions Column -->
-      <v-col>
+      <v-col class="col-auto">
```

Comment:
This makes this column "auto width", which is this case is the minimum width needed for its content. This lets the other column fill the rest of the column.